### ISSUE TITLE
[Feat] Webhook when page is processed

### ISSUE THREAD

**Issue by JakobStadlhuber** [mendableai/firecrawl#214]
**Problem Description**
I find it better if its also possible to get notified when a scrape, crawl process of a page is finished.

**Proposed Feature**
a webhook that is fired everytime a page is completely processed and scraped in the api

**Alternatives Considered**
subscribe to redis

**Implementation Suggestions**
fire in the `convertUrlsToDocuments` the webhook

**Use Case**
an application can react to new scraped pages

**Comment by JakobStadlhuber** [comment]
Should I take a look into it or do you have an existing flow/solution for subscribing to changes without pulling the status api?

**Comment by nickscamara** [comment]
@JakobStadlhuber We have webhooks support for /crawl right now but not /scrape. Have you tried that? We also need to improve the docs around it. I think there is only an explanation of it in the playground get code if I am not mistaking.

**Comment by JakobStadlhuber** [comment]
I see, thanks for mentioning. But I think its not configurable at the moment isn't it?

**Comment by nickscamara** [comment]
Take a look under the account page on the dashboard. There should be a webhook url that you can set there to receive a ping when the crawl job is done. That would be awesome if you can improve the functionality further + have better docs/readme for it! Thank you!

**Comment by JakobStadlhuber** [comment]
Thank you for the clarification. In the self-hosted version is no directly ENV Variable to configure the webhook, probably this [https://github.com/mendableai/firecrawl/blob/260e31c68b880595a76821373b42cb600c9d5dc7/apps/api/src/services/webhook.ts#L4](https://github.com/mendableai/firecrawl/blob/260e31c68b880595a76821373b42cb600c9d5dc7/apps/api/src/services/webhook.ts#L4 ) is the file were optionally ENV Variables can be passed.

**Comment by nickscamara** [comment]
Yes, I believe webhooks are coupled with the database right now. Feel free to pr a decoupled version of it so we can get this working for self host!

**Comment by JakobStadlhuber** [comment]
okay thanks for the info, I will see when I have time. Thank you :)
